plugin/hpsa/hpsa.py

```diff
@@ -502,12 +503,26 @@ def _hp_disk_to_lsm_disk(hp_disk, sys_id, ctrl_num, key_name,
         disk_type = _disk_type_of(hp_disk)
         blk_size = int(hp_disk['Native Block Size'])
         blk_count = int(_hp_size_to_lsm(hp_disk['Size']) / blk_size)
+
+        if 'Disk Name' in hp_disk.keys():
+            regex_match = re.compile("/dev/(sd[a-z]+)").search(disk_sd_path)
```

Comment:
That's a good catch, it exists in the branch that I cherry-picked from, but does not exist here...I'll figure something out.

Are you suggesting pyudev to read sysfs attributes, or for some other reason? I suppose since it's in the repo now as a dependency, it can comfortably be used. When I created this PR though, it wasn't being pulled in.

Thanks for taking a look!